README.md

```diff
@@ -1,20 +1,93 @@
-# Rivet icons
-These are the WIP source files for a basic set of Rivet icons.
+## Rivet icons
+Source files for the Rivet icon set
 
 [**View Demo**](https://indiana-university.github.io/rivet-icons/)
 
-## Basic guidelines
+### Usage
+Inside the `/dist` folder there is a file called `rvt-icons.svg` that a SVG sprite sheet that uses the `<symbol>` element to bundle all of the Rivet icons together into one file. To use the bundled SVGs in your project:
```

Comment:
> Inside the `/dist` folder there is a file called `rvt-icons.svg`

Does this mean that `rivet-icons` will soon be published to the npm registry?

Reply:
That's what I was thinking, but I wonder if we should wait until we work out the custom build script we've been talking about here?

My thought was to publish the `/dist` folder as a package that way anyone using build tools that are capable, could `import` either the `rvt-icons.svg`, or even individual .svgs form the `/dist/svgs` (not committed with source since it's a duplicate of `/src/svg/` folder. I dunno?

Reply:
There's value in going ahead and publishing the update, at least to just secure the npm name. These other refinements with custom sprites could easily come later, even if it is relatively soon.